policyengine_us/variables/gov/states/co/tax/income/credits/cdcc/low_income/co_low_income_cdcc.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class co_low_income_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado Low-income Child Care Expenses Credit"
+    unit = USD
+    documentation = "https://casetext.com/statute/colorado-revised-statutes/title-39-taxation/specific-taxes/income-tax/article-22-income-tax/part-1-general/section-39-22-1195-child-care-expenses-tax-credit-legislative-declaration-definitions"
+    definition_period = YEAR
+    defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.co.tax.income.credits.cdcc.low_income
+        # Filer is eligible if AGI is below $25,000
+        agi = tax_unit("adjusted_gross_income", period)
+        # The childcare expenses can not exceed the earned income of the filer
+        # if filing jointly - the lesser of either individual's earned income
+        earned_income = tax_unit("min_head_spouse_earned", period)
+        # Get sum of all qualified child care expenses
+        expenses = tax_unit.spm_unit("childcare_expenses", period)
+        person = tax_unit.members
+        age = person("age", period)
```

Comment:
check for "is_child" here and make 
child = person("is_child", period)
age_eligible = (age < p.child_age_limit) & is_child